client/me/index.js

```diff
@@ -1,11 +1,18 @@
 import page from '@automattic/calypso-router';
-import { makeLayout, render as clientRender } from 'calypso/controller';
+import { makeLayout, render as clientRender, setSelectedSiteIdByOrigin } from 'calypso/controller';
 import * as controller from './controller';
 
 import './style.scss';
 
 export default function () {
-	page( '/me', controller.sidebar, controller.profile, makeLayout, clientRender );
+	page(
+		'/me',
```

Comment:
We also need the middleware on `/me/account` because that's where the `My Account` menu on the `Howdy, {display_name}` menu links to.

Reply:
https://github.com/Automattic/wp-calypso/pull/92777/commits/7808ad61e44eece2387b4cc92f11dbcd17b97661